ledger/metrics/src/main/scala/com/daml/metrics/VarGauge.scala

```diff
@@ -7,11 +7,13 @@ import java.util.concurrent.atomic.AtomicReference
 
 import com.codahale.metrics.Gauge
 
-case class VarGauge[T](initial: T) extends Gauge[T] {
-  private val ref = new AtomicReference[T](initial)
-  def updateValue(x: T): Unit = ref.set(x)
-  def updateValue(up: T => T): Unit = {
-    val _ = ref.updateAndGet(up(_))
+object Gauges {
```

Comment:
I wish filename is also changed together with the classname

Reply:
Renamed in [this commit](https://github.com/digital-asset/daml/pull/15067/commits/d0aadcf3addc8278d9c34dd1717b3b0541d52574).